### PR TITLE
testing/termite new aport

### PR DESCRIPTION
**Pull request by ianbashford**
Termite: A simple VTE-based terminal
https://github.com/thestinger/termite/

### CODE REVIEW

File: testing/termite/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	pcre2 gnutls"
+subpackages="$pkgname-doc $pkgname-terminfo::noarch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/thestinger/termite/archive/v$pkgver.tar.gz
+        vte-ng-$_vtever.tar.gz::https://github.com/thestinger/vte-ng/archive/$_vtever.tar.gz"
+builddir="${srcdir}/${pkgname}-${pkgver}"
+
+unpack() {
+	default_unpack
+	cd "${pkgname}-${pkgver}"
+	git clone https://github.com/thestinger/util
```
Comment (maxice8) [inline testing/termite/APKBUILD]: can this be added to source= instead ? 

See how telegram-desktop does it to add git-submodules

File: testing/termite/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="x86_64"
+options="!check"  # No test suite.
+license="LGPL"
```
Comment (maxice8) [inline testing/termite/APKBUILD]: This is not an SPDX compliant license identifier

File: testing/termite/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Contributor: Ian Bashford <ianbashford@gmail.com>
+# Maintainer: Ian Bashford <ianbashford@gmail.com>
+pkgname=termite
+pkgver=15
+pkgrel=0
+_vtever=0.50.2-ng
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="x86_64"
```
Comment (maxice8) [inline testing/termite/APKBUILD]: Can't this be done for more arches ?

File: testing/termite/APKBUILD (hunk 1 above)
Comment (ianbashford) [reply testing/termite/APKBUILD]: Done.
Thanks for the pointer;  I was hoping to make that neater.

File: testing/termite/APKBUILD (hunk 2 above)
Comment (ianbashford) [reply testing/termite/APKBUILD]: Done

File: testing/termite/APKBUILD (hunk 3 above)
Comment (ianbashford) [reply testing/termite/APKBUILD]: I've changed it to "all" - I don't see why it shouldn't work, as its a simple app.
The travis CI build only looks like it runs for x86_64 - if there's any other way I can check they all work I'd be happy to look into it.

File: testing/termite/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+_util_commit=409b8449ab51fccf51057621168c9c15c54d4807
+pkgdesc="A simple VTE-based terminal"
+url="https://github.com/thestinger/termite/"
+arch="all"
+options="!check"  # No test suite.
+license="LGPL-2.1-or-later AND MIT"
+depends="$pkgname-terminfo"
+makedepends="intltool libtool automake autoconf gtk-doc gtk+3.0-dev
+	glib-dev pcre2-dev gnutls-dev gperf ncurses
+	pcre2 gnutls"
```
Comment (maxice8) [inline testing/termite/APKBUILD]: don't think pcre2 and gnutls are required when their -dev verions are used

Comment (ianbashford) [reply testing/termite/APKBUILD]: you're right -- removed them.

**Comment by maxice8** [comment]
Merged with a few changes. There were lot of occurrences of space used in indentation instead of tabs. 

Please use apkbuild-lint from atools next time.

**Pull request closed without merging**